test/message-generator/test/pool-sri-test-extended_1.json

```diff
@@ -70,11 +70,11 @@
                             ],
                             [
                                 "coinbase_tx_suffix",
-                                {"Seq064k": [[85, 0],  [255, 255, 255, 255, 1, 0, 242, 5, 42, 1, 0, 0, 0, 67, 65, 4, 70, 109, 127, 202,
-                                229, 99, 229, 203, 9, 160, 209, 135, 11, 181, 128, 52, 72, 4, 97, 120, 121,
-                                161, 73, 73, 207, 34, 40, 95, 27, 174, 63, 39, 103, 40, 23, 108, 60, 100, 49,
-                                248, 238, 218, 69, 56, 220, 55, 200, 101, 226, 120, 79, 58, 158, 119, 208, 68,
-                                243, 62, 64, 119, 151, 225, 39, 138, 172, 0, 0, 0, 0]]}
+
+                                {"Seq064k": [[43, 0], [255, 255, 255, 255, 1, 0, 242, 5, 42, 1, 0,
```

Comment:
unrelated to this PR but any idea why MG parses this a Seq064K instead of B064k?